app/uk/gov/hmrc/fileupload/StreamImplicits.scala

```diff
@@ -14,8 +14,17 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.fileupload.infrastructure
+package uk.gov.hmrc.fileupload
 
-import play.modules.reactivemongo.MongoDbConnection
+import akka.stream.{ActorMaterializer, Materializer}
+import play.api.libs.concurrent.Akka
 
-object DefaultMongoConnection extends MongoDbConnection
+object Streams {
+
+  object Implicits {
+    import play.api.Play.current
```

Comment:
This shouldn't really use Play.current since we have and actor system in the Application Module we should use that